client/matchmaking/admin-matchmaking-times.tsx

```diff
@@ -0,0 +1,690 @@
+  letter-spacing: 0.5px;
+`
+
+const HistoryCard = styled.div`
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  padding: 16px;
+  background-color: rgba(255, 255, 255, 0.05);
+  border-radius: 8px;
+  transition: background-color 0.2s ease;
+
+  &:hover {
+    background-color: rgba(255, 255, 255, 0.08);
+  }
+`
+
+const CurrentMatchmakingCard = styled(HistoryCard)`
+  background-color: rgba(255, 193, 7, 0.1);
+  border: 1px solid rgba(255, 193, 7, 0.2);
+  box-shadow: 0 0 10px rgba(255, 193, 7, 0.1);
+
+  &:hover {
+    background-color: rgba(255, 193, 7, 0.15);
+  }
+`
+
+const HistoryInfo = styled.div`
+  display: flex;
+  flex-direction: column;
+  gap: 4px;
+`
+
+const HistoryDate = styled.div`
+  ${bodyMedium};
+  color: var(--theme-on-surface-variant);
+`
+
+const HistoryStatus = styled.div`
+  display: flex;
+  align-items: center;
+  gap: 8px;
+`
+
+const HistoryActions = styled.div`
+  display: flex;
+  gap: 8px;
+`
+
+const LoadMoreButton = styled(TextButton)`
+  width: 100%;
+  margin: 8px 0;
+`
+
+const EnabledText = styled.span`
+  color: var(--theme-success);
+`
+
+const DisabledText = styled.span`
+  color: var(--theme-error);
+`
+
+const CurrentText = styled.span`
+  color: var(--theme-amber);
+`
+
+const FinishedText = styled.span`
+  color: var(--theme-on-surface-variant);
+`
+
+const EmptyStateContainer = styled.div`
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  justify-content: center;
+  padding: 32px;
+  background-color: rgba(255, 255, 255, 0.03);
+  border-radius: 8px;
+  margin: 16px 0;
+  text-align: center;
+`
+
+const EmptyStateIcon = styled(MaterialIcon)`
+  font-size: 48px;
+  color: var(--theme-on-surface-variant);
+  margin-bottom: 24px;
+  opacity: 0.7;
+`
+
+const EmptyStateText = styled.p`
+  ${bodyMedium};
+  color: var(--theme-on-surface-variant);
+  max-width: 400px;
+  margin: 0;
+`
+
+export function AdminMatchmakingTimes() {
+  const { t } = useTranslation()
+
+  const [activeTab, setActiveTab] = useState<MatchmakingType>(MatchmakingType.Match1v1)
+
+  return (
+    <Container>
+      <Tabs activeTab={activeTab} onChange={setActiveTab}>
+        {ALL_MATCHMAKING_TYPES.map(type => (
+          <TabItem key={type} text={matchmakingTypeToLabel(type, t)} value={type} />
+        ))}
+      </Tabs>
+
+      <PageHeadline>Matchmaking times</PageHeadline>
+
+      {/** Recreate this component when the active tab changes so it clears its local state. */}
+      <AddAndDisplayMatchmakingTimes key={activeTab} activeTab={activeTab} />
+    </Container>
+  )
+}
+
+function AddAndDisplayMatchmakingTimes({ activeTab }: { activeTab: MatchmakingType }) {
+  const dispatch = useAppDispatch()
+  const snackbarController = useSnackbarController()
+
+  const [startDate, setStartDate] = useState('')
+  const [invalidDate, setInvalidDate] = useState(false)
+  const [enabled, setEnabled] = useState(false)
+
+  const [addError, setAddError] = useState<Error>()
```

Comment:
`addError` is a very strange name for this, makes it seem like this is a function for adding errors